src/pages/PokemonsList.js

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchPokemons } from '../store/slicePokemons';
+import { Card } from '../components';
+import { getIdFromPokenUrl } from '../helper/helps';
+import { Modal } from '../components/Modal';
+import './pokemonsList.css';
+
+export function PokemonsList() {
+  const { selected, data_filtered, onClickPokemons, onCloseModal } =
+    useManager();
+
+  return (
+    <div>
+      <Modal visible={!!selected} url={selected} onClose={onCloseModal} />
+      <div className='grid-list'>
+        {data_filtered.map((element, index) => (
+          <Card key={index} name={element.name} url={element.url} onClick={onClickPokemons} />
+        ))}
+      </div>
+    </div>
+  );
+}
+
+const search = (array, text) =>
+  array.filter((element) => !element.name.toLowerCase().indexOf(text.toLowerCase()));
+
+function useManager() {
+  const [selected, setSelect] = useState();
+  const dispatch = useDispatch();
+  const { array, textfilter } = useSelector((state) => state.pokemons);
```

Comment:
lowerCamelCase for constants!!!
```suggestion
  const { array, textFilter } = useSelector((state) => state.pokemons);
```